bodhi-server/pyproject.toml

```diff
@@ -1,3 +1,167 @@
+[tool.poetry]
+name = "bodhi-server"
+version = "5.7.5"
+description = "Bodhi server"
+readme = "README.rst"
+authors = ["Fedora Infrastructure Team"]
+maintainers = ["Fedora Infrastructure Team <infrastructure@lists.fedoraproject.org>"]
+#platforms = ["Fedora", "GNU/Linux"]
+repository = "https://github.com/fedora-infra/bodhi"
+homepage = "https://bodhi.fedoraproject.rog"
+keywords = ["web", "fedora", "pyramid"]
+license = "GPL-2.0-or-later"
+# Possible options are at https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: System :: Software Distribution",
+    "Framework :: Pyramid",
+    "Programming Language :: JavaScript",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
+]
+packages = [
+    { include = "bodhi" },
+    { include = "tests", format = "sdist"}
+]
+include = [
+    { path = "*.txt", format = "sdist" },
+    { path = "*.ini", format = "sdist" },
+    { path = "*.cfg", format = "sdist" },
+    { path = "*.rst", format = "sdist" },
+    { path = "celeryconfig.py", format = "sdist" },
+    { path = "COPYING", format = "sdist"},
+    { path = "bodhi/server/locale/.placeholder", format = "sdist" },
+    { path = "bodhi/*.ico", format = "sdist" },
+    { path = "bodhi/*.png", format = "sdist" },
+    { path = "bodhi/*.css", format = "sdist" },
+    { path = "bodhi/*.gif", format = "sdist" },
+    { path = "bodhi/*.jpg", format = "sdist" },
+    { path = "bodhi/*.pt", format = "sdist" },
+    { path = "bodhi/*.txt", format = "sdist" },
+    { path = "bodhi/*.mak", format = "sdist" },
+    { path = "bodhi/*.mako", format = "sdist" },
+    { path = "bodhi/*.js", format = "sdist" },
+    { path = "bodhi/*.html", format = "sdist" },
+    { path = "bodhi/*.xml", format = "sdist" },
+    { path = "bodhi/*.ttf", format = "sdist" },
+    { path = "bodhi/*.woff", format = "sdist" },
+    { path = "bodhi/*.woff2", format = "sdist" },
+    { path = "bodhi/*.eot", format = "sdist" },
+    { path = "bodhi/*.otf", format = "sdist" },
+    { path = "bodhi/*.svg", format = "sdist" },
+    { path = "bodhi/*.tpl", format = "sdist" },
+    { path = "bodhi/*.rst", format = "sdist" },
+    { path ="apache/*", format = "sdist" },
+    { path = "docs", format = "sdist" },
+        ]
+exclude = ["development.ini", "docs/_build"]
```

Comment:
This doesn't seem to work, the `development.ini` file is in the resulting `tar.gz` :-/

Reply:
Tried with the `path` and seems it was finally excluded.